Week 4 Pull Request

AliannaFormoso opened the pull request:
# Week 4 `Pull request`

AliannaFormoso commented:
# Week 4 `Lessons`
- Dayjs and Sass Blog
1. Installed `vite`, `dayjs`, and `sass`
2. Structured main folders and files
3. Added Html, first styles and js functionalities.

AliannaFormoso commented:
# Week 4 `Lessons`
- Scss specifities
1. Added Scss files with more specific styles

AliannaFormoso commented:
# Week 4 `Lessons`
- Dayjs and Sass Cvs
1. Installed `vite`, `dayjs`, and `sass`
2. Structured main folders and files
3. Added Html, first styles and js functionalities.

AliannaFormoso commented:
# Week 4 `Lessons`
- Scss specifities CV
1. Added Scss files with more specific styles

EzequielCaste commented on student-cvs/alifm-2843/main.mjs:
```diff
@@ -0,0 +1,54 @@
+import relativeTime from "dayjs/plugin/relativeTime.js";
+dayjs.extend(relativeTime);
+
+//* Toggle between adding and removing the "responsive" class to topnav when the user clicks on the icon */
```
```suggestion
//* Toggle between adding and removing the "responsive" class to topnav when the user clicks on the icon */
```

Just as this comment is suggesting, to **Toggle** the class, there is a method precisely for that in the **classList** property:
https://developer.mozilla.org/en-US/docs/Web/API/Element/classList

AliannaFormoso replied:
Hi!
Great! I'll take a look of the article you recommended, and try to improve my code according to it.
Thank you for letting me know about it!

EzequielCaste commented on student-cvs/alifm-2843/index.html:
```diff
@@ -0,0 +1,311 @@
+                </li>
+             
+                <li>
+                    <a href="javascript:void(0);" class="icon">
```
```suggestion
                    <a href="#" class="icon">
```

can be replaced with a simple "#"

EzequielCaste reviewed (approved):
Review of Blog:
- Everything looks fine to me :+1: 
- One suggestion is to start using semantic HTML elements, for example, replace the `div`'s with `article` or `section`
- Check that the links on top, "Home, Beach, Country" don't have a value set to the `href` attribute, this causes the whole page to refresh. You can include a "#" as the value for `href` to avoid this behavior.

Review of CV
- All good (See comments)

You have my approval since your work meets the requirements for Week 4. good job! 🤗

AliannaFormoso commented:
> Review of Blog:
> 
> * Everything looks fine to me 👍
> * One suggestion is to start using semantic HTML elements, for example, replace the `div`'s with `article` or `section`
> * Check that the links on top, "Home, Beach, Country" don't have a value set to the `href` attribute, this causes the whole page to refresh. You can include a "#" as the value for `href` to avoid this behavior.
> 
> Review of CV
> 
> * All good (See comments)
> 
> You have my approval since your work meets the requirements for Week 4. good job! 🤗


Thank you so much for taking the time to review my PR!
 I'll apply all your suggestions since they really improve my work.
I appreciate it!

AliannaFormoso commented:
# Week 4 `Review`
- Applied Reviewer's suggestions
1. Added semantic tags to blog page as well as addresses to the `href` attributes.
2. Modified function to toggle classes on CV page, as well as modified `href` for the icon.

Arzeus17 commented:
Todo óptimo!

AliannaFormoso commented:
> Todo óptimo!

Muito obrigada!

AliannaFormoso commented:
> # Congratulations @AliannaFormoso you did a great job.
> You have successfully completed the exercises for the week. 👏 Well done! Keep up the excellent work! 😁

Thank you 👍🏼
